prow/plugins/approve/approve.go

```diff
@@ -79,8 +80,35 @@ type state struct {
 }
 
 func init() {
-	plugins.RegisterGenericCommentHandler(pluginName, handleGenericCommentEvent, nil)
-	plugins.RegisterPullRequestHandler(pluginName, handlePullRequestEvent, nil)
+	plugins.RegisterGenericCommentHandler(pluginName, handleGenericCommentEvent, helpProvider)
+	plugins.RegisterPullRequestHandler(pluginName, handlePullRequestEvent, helpProvider)
+}
+
+func helpProvider(config *plugins.Configuration, enabledRepos []string) (*pluginhelp.PluginHelp, error) {
+	doNot := func(b bool) string {
+		if b {
+			return ""
+		}
+		return "do not "
+	}
+
+	approveConfig := map[string]string{}
+	for _, repo := range enabledRepos {
+		parts := strings.Split(repo, "/")
+		if len(parts) != 2 {
+			return nil, fmt.Errorf("invalid repo in enabledRepos: %q", repo)
+		}
+		opts := optionsForRepo(config, parts[0], parts[1])
+		approveConfig[repo] = fmt.Sprintf("Pull requests %srequire an associated issue.\nPull request authors %simplicitly approve their own PRs.", doNot(opts.IssueRequired), doNot(opts.ImplicitSelfApprove))
+	}
+	return &pluginhelp.PluginHelp{
+			Description: "The approve plugin implements a pull request approval process that manages the `approved` label and an approval notification comment. Approval is achieved when the set of users that have approved the PR is capable of approving every file changed by the PR. A user is able to approve a file if their username or an alias they belong to is listed in the 'approvers' section of an OWNERS file in the directory of the file or higher in the directory tree.\n\nPer-repo configuration may be used to require that PRs link to an associated issue before approval is granted. It may also be used to specify that the PR authors implicitly approve their own PRs.",
+			WhoCanUse:   "Anyone can use the approval commands, but only users listed as 'approvers' in appropriate OWNERS files will have an effect on the PRs approval state.",
```

Comment:
Use just `Users listed as 'approvers' in appropriate OWNERS files`. Technically, everyone can use all of the commands.

Also, maybe add a link to OWNERS documentation.

Reply:
Done.